testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -6178,6 +6178,34 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("str_to_map('a:1,b:2,c:3', ',',null)");
   }
 
+  /** Tests {@code SUBSTRING_INDEX} function from Spark. */
```

Comment:
You can take a look at other java docs in this file. We normally use format like:
`/** Test case for...`.
IMO, it helps others to locate the corresponding JIRA ticket quickly.

Reply:
Yes, I refer to other test cases in this in this file., but their format is ```/** Tests {@code methodName} function from Spark. */```, for example :
```
  /** Tests {@code STR_TO_MAP} function from Spark. */
  @Test void testStrToMapFunc() {
    final SqlOperatorFixture f0 = fixture();
    f0.setFor(SqlLibraryOperators.STR_TO_MAP);
    f0.checkFails("^str_to_map('a=1,b=2', ',', '=')^",
        "No match found for function signature STR_TO_MAP\\("
            + "<CHARACTER>, <CHARACTER>, <CHARACTER>\\)", false);

    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SPARK);
    f.checkScalar("str_to_map('a=1,b=2', ',', '=')", "{a=1, b=2}",
```

 Not ```/** Test case for...```.  Is there have a clear rule for java docs in Apache Calcite?

Reply:
No specific rules for it. As long as it's clear and helps others to find the corresponding ticket.
What do you think of the following example?
```
/** Test case for
   * <a href="https://issues.apache.org/jira/browse/CALCITE-4861">[CALCITE-4861]
   * Optimisation of chained cast calls can lead to unexpected behaviour.</a>.
   */
```

Reply:
tks